sdk/keyvault/keyvault-admin/src/accessControlModels.ts

```diff
@@ -147,7 +147,7 @@ export interface KeyVaultRoleAssignmentProperties {
   /**
    * The role assignment scope.
    */
-  scope?: KeyVaultRoleScope;
+  roleScope?: KeyVaultRoleScope;
```

Comment:
.NET and Python also uses just `scope`, which is what's defined in swagger. This should not be changed.

Reply:
@vcolin7 is changing Java so you can probably close this PR.

Reply:
Yeah, Heath and I just had a chat about this and will be changing the name in the Properties object to `scope`.

Reply:
Ok ok closing!